src/garage/np/policies/base.py

```diff
@@ -1,68 +1,88 @@
-from garage.core import Parameterized
+"""Base class for policies based on numpy."""
+import abc
 
 
-class Policy(Parameterized):
+class Policy(abc.ABC):
+    """Base classe for policies based on numpy."""
+
     def __init__(self, env_spec):
-        Parameterized.__init__(self)
         self._env_spec = env_spec
 
     # Should be implemented by all policies
-
+    @abc.abstractmethod
     def get_action(self, observation):
+        """
+        Get action sampled from the policy.
+
+        Args:
+            observation (np.ndarray): Observation from the environment.
+        Returns:
+            action (np.ndarray): Action sampled from the policy.
+
+        """
         raise NotImplementedError
 
     def reset(self):
+        """Reset the policy."""
         pass
 
     @property
     def observation_space(self):
+        """Return the observation space of the environment."""
         return self._env_spec.observation_space
 
     @property
     def action_space(self):
+        """Return the action space for the environment."""
         return self._env_spec.action_space
 
     @property
     def recurrent(self):
         """
-        Indicates whether the policy is recurrent.
+        Indicate whether the policy is recurrent.
+
         :return:
         """
         return False
 
     def log_diagnostics(self, paths):
-        """
-        Log extra information per iteration based on the collected paths
-        """
+        """Log extra information per iteration based on the collected paths."""
         pass
 
     @property
     def state_info_keys(self):
         """
-        Return keys for the information related to the policy's state when
-        taking an action.
+        Get keys describing policy's state.
+
+        Return keys for the information related to the policy's state
+        when taking an action.
+
         :return:
         """
         return list()
 
     def terminate(self):
-        """
-        Clean up operation
-        """
+        """Clean up operation."""
         pass
 
 
 class StochasticPolicy(Policy):
+    """Base class for stochastic policies implemented in numpy."""
+
     @property
     def distribution(self):
         """
+        Get the distribution of the policy.
+
         :rtype Distribution
         """
         raise NotImplementedError
 
+    @abc.abstractmethod
     def dist_info_sym(self, obs_var, state_info_vars):
         """
         Return the symbolic distribution information about the actions.
+
```

Comment:
please remove these old `:param:` style documentations and replace them with the Google style.